docs/connectors.md

```diff
@@ -0,0 +1,40 @@
+## Connector Maintainers
+
+* **Official** -  Official connectors and tools are built, maintained, supported, and tested by Meltano.
+* **Partner** - Partner connectors and tools are built, maintained, supported, and tested by a partner organization of Meltano, including consultancies and other vendors.
+* **Community** - Community connectors and tools are built and maintained by the wider open source community.
+
+## Connector Quality
+
+* **Gold** - Gold connectors are built on the Meltano SDK, are actively used by many Meltano Cloud and open source users, and have broad coverage of data and features.
+* **Silver** - Silver connectors may be built on the Meltano SDK, but their open source and Meltano Cloud usage may be lower compared to a Gold connector.
```

Comment:
@tayloramurphy So these are 1) SDK-based connectors with low to medium usage data (as SDK+high = Gold) and 2) non-SDK connectors with medium to high usage data (as non-SDK+low = Bronze)? I think that makes sense -- even with low usage data, we generally trust a SDK-based tap more than a non-SDK one. I think that can be more clear though:

```suggestion
* **Silver** - Connectors built on the Meltano SDK with low to medium Meltano Cloud and self-managed usage, and connectors not built on the Meltano SDK with high usage.
```

Reply:
@DouweM Yes - you've got the summary mostly right. Many of the transferwise connectors should be Silver - non-SDK based but high usage. An SDK-based connector _could_ be Silver depending on usage and coverage of streams. But that can be adjusted based on our judgement.